extension/src/popup/components/account/AccountHistory.tsx

```diff
@@ -154,20 +156,21 @@ const HistoryItem = ({
       }}
     >
       <HistoryColumnEl>
-        <TimestampEl>
-          {new Date(Date.parse(createdAt)).toLocaleString()}
-        </TimestampEl>
+        <HistoryColumnRowEl>
+          <TimestampEl>
+            {new Date(Date.parse(createdAt)).toLocaleString()}
+            {operationCount > 1 && !isPaymentOperation
+              ? ` • ${operationCount - 1} ops`
+              : null}
+          </TimestampEl>
+        </HistoryColumnRowEl>
         <HistoryColumnRowEl>
           {isPaymentOperation
             ? `${isRecipient ? "Received" : "Sent"} ${operationString}`
             : operationString}
-          {operationCount > 1 && !isPaymentOperation
-            ? ` + ${operationCount - 1} ops`
-            : null}
         </HistoryColumnRowEl>
       </HistoryColumnEl>
       <PaymentColumnEl>{renderPaymentComponent()}</PaymentColumnEl>
-      <OpenExternalIconEl src={IconOpenExternal} alt="open in stellar.expert" />
```

Comment:
We actually still need the ability to open the Operation information in stellar.expert. Can you restore this?

Reply:
I'll fix this in a follow up PR